yarn.lock

```diff
@@ -3,36 +3,42 @@
 
 
 "@babel/code-frame@^7.0.0":
-  version "7.16.7"
-  resolved "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.16.7.tgz"
-  integrity sha512-iAXqUn8IIeBTNd72xsFlgaXHkMBMt6y4HJp1tIaK465CWLT/fG1aqB7ykr95gHHmlBdGbFeWWfyB4NJJ0nmeIg==
+  version "7.22.10"
```

Comment:
I'm not sure how the yarn.lock got changed because it is supposed to only update when there are dependency changes, meaning package.json was changed. I checked out your branch and ran from my side with yarn install, and didn't update the yarn.lock. So, I guess we don't need to worry about it now but keep an eye out if next time your yarn.lock still gets updated despite no dependency changes. We might be able to find a pattern. 

Here is the link talking about yarn.lock: https://github.com/yarnpkg/yarn/issues/4379